docs/topics/api/accounts.rst

```diff
@@ -73,6 +73,55 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 .. http:get:: /api/v3/accounts/profile/
 
 
+----
+Edit
+----
+
+.. _`account-edit`:
+
+.. note::
+    This API requires :doc:`authentication <auth>` and `Users:Edit`
+    permission to edit accounts other than your own.
+
+This endpoint allows some of the details for an account to be updated.  Any fields
+in the :ref:`account <account-object>` (or :ref:`self <account-object-self>`)
+but not listed below are not editable and will be ignored in the patch request.
+
+.. http:patch:: /api/v3/accounts/account/(int:user_id)/
+
+    .. _account-edit-request:
+
+    :<json string|null biography: More details about the user.  No links are allowed.
+    :<json string|null display_name: The name chosen by the user.
+    :<json string|null homepage: The user's website.
+    :<json string|null location: The location of the user.
+    :<json string|null occupation: The occupation of the user.
+    :<json string|null username: username to be used in the account url.  The username can only contain letters, numbers, underscores or hyphens. All-number usernames are prohibited as they conflict with user-ids.
+
+
+-------------------
+Uploading a picture
+-------------------
+
+To upload a picture for the profile the request must be sent as content-type `multipart/form-data` instead of JSON.
+Images must be either PNG or JPG and the maximum file size is 4MB.
+Other :ref:`editable values <account-edit-request>` can be set at the same time.
```

Comment:
Still not 100% convinced because I hate multipart, but enough to drop the argument :)